### PR TITLE
Master

### PR DESCRIPTION
**Pull request by robotechredmond**
Updated to include 4 fixes: 
1) sqlSubnetRef for ILB in azuredeploy.json
2) Default ILB IP address in azuredeploy.json
3) Dependencies on SQLAGListener and SQLAQDatabase resources in CreateFailoverCluster.ps1
4) Added DSC resource to disable automatic storage clustering in CreateFailoverCluster.ps1

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@robotechredmond__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by singhkay** [comment]
@robotechredmond For CLA, if you're an MSFT FTE, can you please send me an email offline so I can remove this check.

**Comment by azurecla** [comment]
__@robotechredmond__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

File: sql-server-2014-alwayson-dsc/azuredeploy.json [hunk 1]
```diff
@@ -194,7 +194,7 @@
       "metadata": {
         "description": "The location of resources that the script is dependent on such as linked templates and DSC modules"
       },
-      "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/sql-server-2014-alwayson-dsc"
+      "defaultValue": "https://raw.githubusercontent.com/robotechredmond/azure-quickstart-templates/master/sql-server-2014-alwayson-dsc"
```
Comment (singhkay) [inline sql-server-2014-alwayson-dsc/azuredeploy.json]: This should point to Azure location

File: sql-server-2014-alwayson-dsc/metadata.json [hunk 2]
```diff
@@ -3,6 +3,6 @@
   "description": "This template creates five new Azure VMs, with a public IP address, NAT'd RDP port, external and internal load balancer and VNet, it configures two VMs to be AD DCs for a new Forest and Domain, and creates a Windows
 	Cluster with the other three VMs, two VMs in the cluster are running SQL Server 2014 and are configured with an availability group and an availability group listener, the other VM is a File Share Witness for the cluster  ",
   "summary": "This template creates 5 Azure VMs with Active Directory and SQL Server Always On",
-  "githubUsername": "simongdavies",
-  "dateUpdated": "2015-06-03"
+  "githubUsername": "robotechredmond",
```
Comment (singhkay) [inline sql-server-2014-alwayson-dsc/metadata.json]: Please leave the original author intact

**Comment by singhkay** [comment]
@simongdavies Can you review this change?

File: sql-server-2014-alwayson-dsc/azuredeploy.json (hunk 1 above)
Comment (robotechredmond) [inline sql-server-2014-alwayson-dsc/azuredeploy.json]: Done.

Comment (simongdavies) [inline sql-server-2014-alwayson-dsc/azuredeploy.json]: This doesn't seem to have been done, I can see that the deploy to Azure button has been changed back but not the default value for this parameter

**Comment by simongdavies** [comment]
Once #discussion_r44381618 and #commitcomment-14296397
have been addressed the this is good to merge

File: sql-server-2014-alwayson-dsc/azuredeploy.json (hunk 1 above)
Comment (robotechredmond) [inline sql-server-2014-alwayson-dsc/azuredeploy.json]: Done - commited change for defaultValue

**Pull request merged**